### DISCUSSION TITLE
content conflicts when nfpm pkg

### DISCUSSION THREAD

**Discussion by welove42** [coder/code-server#5810] [Q&A] [answered]
I follow the documentation steps to yarn package, while going to package in nfmp steps,there throw some error about content collision
```
done (release-packages/code-server-4.9.0-linux-amd64)
Building deb
name: "code-server"
arch: "amd64"
platform: "linux"
version: "v"
guessing packager from target file extension...
using deb packager...
cannot add "release-standalone/node_modules/request/LICENSE" because "release-standalone/node_modules/fs.realpath/LICENSE" is already present at the same destination (/usr/lib/code-server/LICENSE): content collision
error Command failed with exit code 1.
```
This seems to be because nfpm moved the folder inside file to the coder-server directory instead of moving the entire folder to the corresponding directory

<img width="695" alt="image" src="https://user-images.githubusercontent.com/77881340/203515987-3ad8fe1d-44cd-4af1-92c4-b43c752d3da9.png">
I try to change nfpm.yaml and yarn package,It seems to pkg normally
but i'm not sure if it is right

**Comment by lg-nineoneninethreenine** [comment]
Wait are you trying to say that if there is an error in builds, it was due to a forward slash and you fixed it!? I think you have to PR it here if it's not added yet.

I think it is right, what you did. The slash could have been a test but wasn't removed in time. Codes are long to analyse, it happens. Maybe you should open a PR and _perhaps_ pick up other similar errors and fix them, so that many things can be solved.

**Reply by jsjoeio** [reply]
I haven't seen this yet but thank you for raising it! I don't have time to dig in right this moment but feel free to raise a PR!

**Comment by jsjoeio** [comment, marked as answer]
I'm running into the same issue so I'm going to test your fix: https://github.com/coder/code-server/issues/5874#issuecomment-1349264673

**Reply by welove42** [reply]
Thanks for your reply
